common/config.go

```diff
@@ -78,9 +67,6 @@ func LoadFromCommandLine(config *Config) (*Config, error) {
 
 	flagSet.StringVar(&config.ConfigFile, "conf-file", "", "File path of configuration file")
 
-	flagSet.StringVar(&config.PluginDir, "plugin-dir", config.PluginDir, "Docker plugin directory where driver socket is created")
-	flagSet.StringVar(&config.DriverName, "driver-name", config.DriverName, "Name of Infoblox IPAM driver")
```

Comment:
Is it not possible to run multiple instances of the plugin as different ipam-driver anymore?